data/onPostBuild/llmstxt.ts

```diff
@@ -89,7 +89,7 @@ export const onPostBuild: GatsbyNode['onPostBuild'] = async ({ graphql, reporter
     const { title, meta_description } = meta;
 
     try {
-      const url = prefixPath({ url: slug, siteUrl, pathPrefix: basePath });
+      const url = prefixPath({ url: `/docs/${slug}`, siteUrl, pathPrefix: basePath });
```

Comment:
Was gonna suggest that this prefix be part of `basePath`, but it doesn't appear to be something we can control